webviz_config/templates/webviz_template.py.jinja2

```diff
@@ -107,12 +107,6 @@ else:
                 className="styledButton",
               {%- endif -%}
                 children=[
-                  {%- if loop.first -%}
-                    html.Div(
-                        id="dash-issue-#1010",
-                        style={"display":"none"}, 
-                        children=dash_table.DataTable()),
-                  {%- endif -%}
```

Comment:
In #152 it was also [introduced](https://github.com/equinor/webviz-config/pull/152/files#diff-b5cac186109f24b6fad1fd3d8de0318fR17) a `import dash_table` which we perhaps also can remove now.